src/model.js

```diff
@@ -30,6 +32,16 @@ module.exports = function(koop) {
         this.esConfig = config.esConnections[esId];
         const indexConfig = this.esConfig.indices[serviceName];
         let extent = indexConfig.extent;
+        let customSymbolizer = undefined;
+
+        if(indexConfig.customSymbolizer){
+            this.customSymbolizers.forEach(symbolizer => {
+                if(symbolizer.name === indexConfig.customSymbolizer){
+                    customSymbolizer = symbolizer;
+                    return;
+                }
+            });
+        }
```

Comment:
This should be extracted to a helper function `getCustomSymbolizer` so that we can try to maintain the level of abstraction in getData.  With such a helper, the reader doesn't need to digest the details of how we get the custom symbolizer in order to know that is what's happening.  In general, having to use `let` is sometimes a clue that there is opportunity to move code into another function.

Reply:
Done.  Good idea!